server/app/api/speak.py

```diff
@@ -0,0 +1,30 @@
+"""
+Speak API route handlers. They handle requests related to reference text, recording, receiving audio file and so on.
```

Comment:
Do we need another function in this api to store collected audio data to database?

Reply:
A db_helper.py file is added to handle the db operations. The routers will use the helper